settings/src/components/webauthn/register-key.js

```diff
@@ -1,84 +1,109 @@
 /**
  * WordPress dependencies
  */
-import { Button, Spinner, TextControl } from '@wordpress/components';
+import { Button, Notice, Spinner, TextControl } from '@wordpress/components';
 import { useCallback, useContext, useState } from '@wordpress/element';
-import { Icon, check } from '@wordpress/icons';
+import { Icon, check, cancelCircleFilled } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import { GlobalContext } from '../../script';
+import { preparePublicKeyCreationOptions, preparePublicKeyCredential } from './sjinks-utilities';
+import { refreshRecord } from '../../utilities';
 
 /**
  * Render the form to register new security keys.
  *
  * @param {Object}   props
- * @param {Function} props.onRegisterSuccess
+ * @param {Function} props.onCancel
+ * @param            props.onSuccess
```

Comment:
I remember only TypeScript can automatically infer from the context that this field is a {Function}, I might be wrong, though.

Reply:
There is `@callback`, but that seems kind of excessive, since you have to add a docblock at the function definition in addition to the `@param` here. 

That's not a big deal for things like `onRegisterSuccess` where we already have a docblock, but if we wanted to be consistent then we'd also need to do it for small closures like `onCancel`:

https://github.com/WordPress/wporg-two-factor/blob/d633ba92c06f372bc84f8d8f2215010ec6675eae/settings/src/components/webauthn/webauthn.js#L77

Reply:
I was only trying to point out that the param type is empty for `props.onSuccess`, like `onCancel`, it's supposed to be `{Function}` as well. This is just minor stuff, in case one day we want to leverage the JSDoc with some tools, and normally the empty here would make tools not able to parse correctly.

Reply:
Ah, I missed that, thanks! Added in ff3684a 👍🏻